app/lib/widgets/phone_widget.dart

```diff
@@ -76,50 +78,79 @@ class PhoneAlertDialogState extends State<PhoneAlertDialog> {
   Widget build(BuildContext context) {
     return CustomDialog(
         image: Icons.phone,
-        title: 'Add phone number',
+        title: widget.newPhone
+            ? 'Add phone number'
+            : 'Change phone number',
         widgetDescription: SizedBox(
-          height: 100,
-          child: Row(
-            children: <Widget>[
-              Expanded(
-                child: Padding(
-                  padding: const EdgeInsets.symmetric(horizontal: 16),
-                  child: IntlPhoneField(
-                    initialCountryCode: widget.defaultCountryCode,
-                    decoration: const InputDecoration(
-                      labelText: 'Phone Number',
-                      border: OutlineInputBorder(
-                        borderSide: BorderSide(),
+          height: widget.newPhone ? 100 : 180,
+          child: Column(
+            children: [
+              if (!widget.newPhone)
+                Row(
+                  children: <Widget>[
+                    Expanded(
+                      child: Padding(
+                        padding: const EdgeInsets.symmetric(horizontal: 16),
+                        child: Text(
+                          'Changing your phone will require you to go through the phone verification process again.',
```

Comment:
```suggestion
                          'Changing your phone will require re-verification.',
```

Same with email msg.

Also, can we add more space between the msg & the input? and maybe increase space at the bottom of the cancel btn.

Reply:
- All applied except the space under `Cancel` button, It's a custom dialog used everywhere. They should be all consistent.